projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -202,39 +210,40 @@ export class UpdateChanges {
                 continue;
             }
 
-            let base;
-            let replace;
+            let base: string;
+            let replace: string;
             let groups = 1;
             let searchPattern;
 
             if (type === BindingType.output) {
-                base = String.raw`\(${change.name}\)=(["'])`;
-                replace = `(${change.replaceWith})=$1`;
+                base = String.raw`\(${change.name}\)=(["'])(.*?)\1`;
+                replace = `(${change.replaceWith})=$1$2$1`;
             } else {
                 // Match both bound - [name] - and regular - name
-                base = String.raw`(\s\[?)${change.name}(\s*\]?=)(["'])`;
-                replace = String.raw`$1${change.replaceWith}$2$3`;
+                base = String.raw`(\s\[?)${change.name}(\s*\]?=)(["'])(.*?)\3`;
+                replace = String.raw`$1${change.replaceWith}$2$3$4$3`;
                 groups = 3;
             }
 
             let reg = new RegExp(base, 'g');
             if (change.remove || change.moveBetweenElementTags) {
                 // Group match (\1) as variable as it looks like octal escape (error in strict)
-                reg = new RegExp(String.raw`\s*${base}.*?${'\\' + groups}(?=\s|\>)`, 'g');
+                reg = new RegExp(String.raw`\s*${base}.*?(?=\s|\>)`, 'g');
```

Comment:
Okay this cannot be right, this add the "search till end of value" (`.*?`) based on the old base regex, but that's now added in the base as well. That should result in some pretty wonky expression that can match something after the closing quote mark or even still is doing a lot more pointless lookups.
This separate regex for the deletion shouldn't be needed anymore.

Reply:
Will leave this as is for the moment, as it seems to capture some edge cases (leading/trailing spaces). Could investigate in the future.